Modules/Devices/test/Devices.Domain.Tests/PushNotifications/CreateRandomIdentifierForPushNotificationRegistration.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Backbone.Modules.Devices.Domain.Aggregates.PushNotifications;
+using Backbone.Modules.Devices.Domain.Aggregates.PushNotifications.Handles;
+using FluentAssertions;
+using Xunit;
+using static Backbone.UnitTestTools.Data.TestDataGenerator;
+using Environment = Backbone.Modules.Devices.Domain.Aggregates.PushNotifications.Environment;
+
+namespace Backbone.Modules.Devices.Domain.Tests.PushNotifications;
+
+public class CreateRandomIdentifierForPushNotificationRegistration
+{
+    [Fact]
+    public void Generate_DevicePushIdentifier_while_instancing_PnsRegistration()
```

Comment:
This test should test all properties of the PnsRegistration, and not just the DevicePushIdentifier. It would be too tedious to write a separate test for each property.